performance_tests/README.md

```diff
@@ -0,0 +1,134 @@
+# Performance Tests for ESDK Python
+
+## License
+
+This project is licensed under the Apache-2.0 License.
+
+## Overview
+
+Here are the keyrings / master key-providers that we plan to test:
```

Comment:
This is fine for now -- once you're done, let's update this README with what "was done"